client/client_test.go

```diff
@@ -231,7 +235,7 @@ func TestClientRunTransaction(t *testing.T) {
 
 	for _, commit := range []bool{true, false} {
 		value := []byte("value")
-		key := []byte(fmt.Sprintf("key-%t", commit))
+		key := []byte(fmt.Sprintf("%s/key-%t", testUser, commit))
```

Comment:
It's weird that this test uses the user name as the key prefix; I would rather have a distinct `testDatabase` (`testNamespace`? `testPrefix`?) to emphasize that they are conceptually different.